src/components/Dropdown/DropdownNativeSelect.st.css

```diff
@@ -0,0 +1,165 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color;
+}
+
+:import {
+    -st-from: "../Button/Button.st.css";
+    -st-default: TPAButton;
+}
+
+:import {
+    -st-from: "../Input/Input.st.css";
+    -st-default: TPAInput;
+    -st-named: backgroundColor, colorDisabled, colorPlaceholder, color as inputColor, errorColor, borderColor;
+}
+
+:import {
+    -st-from: "wix-ui-core/index.st.css";
+    -st-named: Tooltip;
+}
+
+:import {
+    -st-from: "../Tooltip/Tooltip.st.css";
+    -st-default: TPATooltip;
+}
+
+:vars {
+    BorderHoverColor: color-5;
+}
+
+:vars {
+    borderHoverColor: color(value(BorderHoverColor));
+}
+
+.wrapper {
+    position: relative;
+    min-width: 68px;
+    background-color: value(backgroundColor);
+}
+
+.root {
+    -st-extends: TPAButton;
+    -st-mixin: TPAButton(
+            MainBackgroundColor 'value(backgroundColor)',
+            MainBorderWidth '1px',
+            MainBorderColor 'value(borderColor)',
+            MainTextColor 'value(inputColor)'
+    );
+    -st-states: disabled, error, placeholder, rtl, icon;
+    display: flex;
+    cursor: pointer;
+    box-sizing: border-box;
+    outline: none;
+    appearance: none;
+}
+
+.root:mobile:size(medium) {
+    padding: 8px 12px;
+    background-color: transparent;
+}
+
+.root:mobile:size(medium):icon {
+    padding: 8px 36px;
+}
+
+.root:mobile:rtl {
+    /*background: value(backgroundColor) no-repeat right 13px center url("../../assets/icons/CaretDown.svg");*/
+    background-image: url("../../assets/icons/CaretDown.svg");
```

Comment:
Do we use this method only in `rtl`?
Didn't we say we can use an absoluted div instead?

Reply:
the absolute div replaces this css, forgot to delete it.
(another) nice catch

Reply:
Still needs to be deleted no? 🤔